### PR TITLE
add license information

### PR DESCRIPTION
**Pull request by weissi**
this adds the license information. Add the [`README.md`](https://github.com/weissi/swift-server-logging-api-proposal/blob/master/README.md) which is the proposal always said, the code will be released under the Apache 2 license.

I did however forget to add a license file / license header to all files so to be sure every contributor is required to acknowledge that they are happy with this.

If you are one of the contributors, please confirm in a comment that you're happy with this.

CC @ianpartridge @ianpartridge @ktoso @tomerd @moiseev @hartbit @tkrajacic 

I've left a mention for you all next to your name in the CONTRIBUTORS file.

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+by employees of that company is held by the company itself, only the company
+needs to be listed here.
+
+## COPYRIGHT HOLDERS
+
+- Apple Inc. (all contributors with '@apple.com')
+
+### Contributors
+
+- David Hart <david@hartbit.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:14]: @hartbit please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`).

Comment (weissi) [inline CONTRIBUTORS.txt]: @iainsmith please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Iain Smith <iainsmith@users.noreply.github.com>
+- Ian Partridge <i.partridge@uk.ibm.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:16]: @ianpartridge  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- Johannes Weiss <johannesweiss@apple.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:17]: @weissi  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- Konrad `ktoso` Malawski <konrad.malawski@project13.pl>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @ktoso  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- Max Moiseev <moiseev@apple.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:19]: @moiseev  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- Tanner <me@tanner.xyz>
```
Comment (weissi) [inline CONTRIBUTORS.txt:20]: @tanner0101  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+- Thomas Krajacic <tkrajacic@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:21]: @tkrajacic please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+- Tomer Doron <tomerd@apple.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt]: @tomerd  please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt (hunk 6 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: I can confirm that I am happy for my contributions to be licensed under the Apache 2 license.

File: CONTRIBUTORS.txt [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+needs to be listed here.
+
+## COPYRIGHT HOLDERS
+
+- Apple Inc. (all contributors with '@apple.com')
+
+### Contributors
+
+- David Hart <david@hartbit.com>
+- Iain Smith <iainsmith@users.noreply.github.com>
```
Comment (weissi) [inline CONTRIBUTORS.txt:15]: @iainsmith please confirm in a comment that you are happy with your code being added to a Swift Logging API package under the terms of the Apache 2 license (see `LICENSE.txt`)

File: CONTRIBUTORS.txt (hunk 7 above)
Comment (tkrajacic) [reply CONTRIBUTORS.txt:21]: More than happy to have any contributions of mine added. Proud even ;)

File: CONTRIBUTORS.txt (hunk 6 above)
Comment (tanner0101) [reply CONTRIBUTORS.txt:20]: I approve, thanks @weissi :)

File: CONTRIBUTORS.txt (hunk 5 above)
Comment (moiseev) [reply CONTRIBUTORS.txt:19]: Confirmed.

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (hartbit) [reply CONTRIBUTORS.txt:14]: I am happy with my code being added to a Swift Logging API package under the terms of the Apache 2.

File: CONTRIBUTORS.txt (hunk 4 above)
Comment (ktoso) [reply CONTRIBUTORS.txt]: Confirmed 👍

Comment (ktoso) [reply CONTRIBUTORS.txt]: Please see this PR too https://github.com/weissi/swift-server-logging-api-proposal/pull/53 with added mail mappings for myself

File: CONTRIBUTORS.txt (hunk 8 above)
Comment (tomerd) [reply CONTRIBUTORS.txt]: can we use `tomer@apple.com`?

File: .mailmap [hunk 10]
```diff
@@ -0,0 +1,5 @@
+Tomer Doron <tomerd@apple.com> <tomer.doron@gmail.com>
+Tomer Doron <tomerd@apple.com>
+Tomer Doron <tomerd@apple.com> tomer doron <tomer@apple.com>
```
Comment (tomerd) [inline .mailmap]: collapse?

File: CONTRIBUTORS.txt (hunk 4 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: Merged

File: CONTRIBUTORS.txt (hunk 3 above)
Comment (weissi) [reply CONTRIBUTORS.txt:17]: I can confirm that I am happy for my contributions to be licensed under the Apache 2 license.

File: .mailmap (hunk 10 above)
Comment (weissi) [reply .mailmap]: I think we can't collapse more as you used quite a few different spellings and emails

File: CONTRIBUTORS.txt (hunk 8 above)
Comment (weissi) [reply CONTRIBUTORS.txt]: done

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (ianpartridge) [reply CONTRIBUTORS.txt:16]: I am happy with my code being added to a Swift Logging API package under the terms of the Apache 2.

File: .mailmap (hunk 10 above)
Comment (tomerd) [reply .mailmap]: that the way i roll

File: CONTRIBUTORS.txt (hunk 8 above)
Comment (tomerd) [reply CONTRIBUTORS.txt]: confirmed and thanks

File: CONTRIBUTORS.txt (hunk 9 above)
Comment (iainsmith-snippets) [reply CONTRIBUTORS.txt:15]: Happy for my contributions to this repo to be added under the Apache 2 license.

**Comment by weissi** [comment]
Ok, we’ve got all!

**Pull request merged**